app/routes/profile/$email/index.tsx

```diff
@@ -114,67 +171,204 @@ export const ProfileInfo = () => {
+                  <Box
+                    sx={{ display: "flex", flexDirection: "row", gap: "8px" }}
+                  >
+                    <BusinessIcon />
+                    <Typography>{profileData.businessUnit}</Typography>
+                  </Box>
+                  <Box
+                    sx={{ display: "flex", flexDirection: "row", gap: "8px" }}
+                  >
+                    <PlaceIcon />
+                    <Typography>{profileData.location}</Typography>
                   </Box>
+                  {isEditGithubUserActive ? (
+                    <ValidatedForm
+                      id="updateGithubUserForm"
+                      method="post"
+                      subaction="UPDATE_GITHUB_USER"
+                      validator={githubUserValidator}
+                      defaultValues={{
+                        githubUser: profileData.githubUser || "",
+                      }}
+                    >
+                      <Box
+                        sx={{
+                          display: "flex",
+                          flexDirection: "row",
+                          gap: "8px",
+                          alignItems: "center",
+                        }}
+                      >
+                        <GitHubIcon />
+                        <TextField
+                          variant="outlined"
+                          name="githubUser"
+                          size="small"
+                          defaultValue={profileData.githubUser}
+                          {...getInputProps({ id: "githubUser" })}
+                        />
+                        <IconButton
+                          type="submit"
+                          onClick={() => {
+                            handleUpdateGithubUser();
+                          }}
+                        >
+                          <SaveIcon />
+                        </IconButton>
+                      </Box>
+                      {githubUsererror && (
+                        <p style={{ color: "red" }}>{githubUsererror}</p>
+                      )}
+                    </ValidatedForm>
+                  ) : (
+                    <Box
+                      sx={{
+                        display: "flex",
+                        flexDirection: "row",
+                        gap: "8px",
+                        alignItems: "center",
+                      }}
+                    >
+                      <GitHubIcon />
+                      <Typography>
+                        {profileData.githubUser?.length
+                          ? profileData.githubUser
+                          : "<Not specified>"}
+                      </Typography>
+                      <IconButton
+                        type="button"
+                        onClick={() => {
+                          setIsEditGithubUserActive(true);
+                        }}
+                      >
+                        <EditSharp />
```

Comment:
maybe we should only allow edit to admins and themselves

Reply:
Fixed!